app/models/concerns/helpable.rb

```diff
@@ -57,10 +53,13 @@ def register_url
 
       if [200, 201].include?(response.status)
         # update minted column after first successful registration in handle system
-        self.update_attributes(minted: Time.zone.now, updated: Time.zone.now) if minted.blank?
+        success = true
```

Comment:
shouldn't this be `success = false`?

Reply:
No because as far as I can tell the update_attribute will only return false if there is a problem therfore the default is passing unless it tells you otherwise.